src/platforms/stm32f4-discovery.c

```diff
@@ -989,14 +989,14 @@ uint32_t cycle_count() {
 static volatile int adc_gather_in_progress = 0;
 void adc_gather() {
 #ifdef SPI_TLC2543
-  static uint16_t spi_tx_list[] = {
+  static const uint16_t spi_tx_list[] = {
```

Comment:
If making this const keeps it in flash instead of sram, can the dma still read it?